src/main/java/nextstep/subway/application/LineService.java

```diff
@@ -0,0 +1,63 @@
+package nextstep.subway.application;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.LineRepository;
+import nextstep.subway.domain.Station;
+import nextstep.subway.dto.LineRequest;
+import nextstep.subway.dto.LineResponse;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+
+    private final LineRepository lineRepository;
+    private final StationService stationService;
+
+    public LineService(LineRepository lineRepository, StationService stationService) {
+        this.lineRepository = lineRepository;
+        this.stationService = stationService;
+    }
+
+    @Transactional
+    public LineResponse saveLine(LineRequest lineRequest) {
+        Station upStation = stationService.findById(lineRequest.getUpStationId());
+        Station downStation = stationService.findById(lineRequest.getDownStationId());
+        Line line = lineRepository.save(Line.of(lineRequest, upStation, downStation));
+
+        return LineResponse.of(line);
+    }
+
+    public List<LineResponse> findAllLines() {
+        List<Line> lines = lineRepository.findLineAndStations();
+        return lines.stream().map(LineResponse::of).collect(Collectors.toList());
```

Comment:
```suggestion
        return lines.stream()
	        .map(LineResponse::of)
	        .collect(Collectors.toList());
```
중요한 건 아니지만 stream 쓸 때 보통 이렇게 개행을 하곤 합니다 : )

Reply:
감사합니다! 제 구글 java 스타일 포맷을 사용하는데 설정이 잘못되있나보네요 ㅠ